src/index.html

```diff
@@ -1,12 +1,310 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="menu">
+      <a class="img" href="#home"></a>
+      <nav class="nav">
+        <a class="nav__list is-active">APPLE</a>
+        <a class="nav__list">SAMSUNG</a>
+        <a class="nav__list">Smartphones</a>
+        <a class="nav__list" data-qa="nav-hover">Laptops & Computers</a>
+        <a class="nav__list">Gadgets</a>
+        <a class="nav__list">Tablets</a>
+        <a class="nav__list">Photo</a>
+        <a class="nav__list">Video</a>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+            class="card__img"
+           alt="APPLE iMac Retina 5K Monoblock"
+           src="images/imac.jpeg"
+        >
+        <p class="card__name">
+           APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__product-code">
+           Product code: 195434
+        </p>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="rewiew">Reviews: 5</p>
+        </div>
+
+        <div class="card__reviews price">
+          <p class="text">Price:</p>
+          <p class="number">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+        Buy
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add a blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```